src/style.css

```diff
@@ -0,0 +1,112 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__content,
+.card__title {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
```

Comment:
Do you need these properties? Block elements will be placed on the new line by default